crypto/nrf_cc312_platform/include/nrf_cc3xx_platform.h

```diff
@@ -90,6 +90,24 @@ bool nrf_cc3xx_platform_rng_is_initialized(void);
  */
 void CRYPTOCELL_IRQHandler(void);
 
+
+/** @brief Function to get the nonce seed used for encrypted ITS usage
+ *
+ * @param[in]	buffer 	Buffer to fill the nonce seed generated during boot.
+ *
+ * @return Zero on success, otherwise a non-zero error code.
+ */
+int nrf_cc3xx_platform_get_nonce_seed(uint8_t buffer[8]);
```

Comment:
The buffer size here (and for `nrf_cc3xx_plaform_get_boot_seed()`) should have a define IMO, so it's easy to create correctly-sized arrays.

Reply:
This will need changes in the runtime library so I will create a task for the next version. The 0.9.14 is now merged.

Reply:
Agreed, this was an oversight on my utilizing a magic number. Will be rectified in the next version of the runtime library